internal_types/src/entry.rs

```diff
@@ -445,6 +445,37 @@ impl<'a> Column<'a> {
             .expect("name must be present in flatbuffers Column")
     }
 
+    pub fn inner(&self) -> &entry_fb::Column<'a> {
+        &self.fb
+    }
+
+    pub fn influx_type(&self) -> InfluxColumnType {
```

Comment:
This encodes the mapping from the Entry's types to the Influx data model types used by IOx's schema plumbing. As mentioned in the PR description, once we support types outside the InfluxDB data model we may want to expand either this enum or create a new parent enumeration, but this should be sufficient for now?

Reply:
Maybe you could make the API more "future proof" by making this return an `Option` - similar to how `Schema` handles fields in the underlying `arrow::Schema` that don't have a corresponding type.

```suggestion
    pub fn influx_type(&self) -> Option<InfluxColumnType> {
```

That way people who use this API  will now they should be prepared to handle non `InfluxColumnTypes` in the future

Reply:
At the moment this would just move the panic, as the MUB doesn't know how to handle a type outside the Influx data model. Unless you feel strongly I'd be inclined to leave it as is and revisit once that story is more fleshed out?